### PR TITLE
vview.py: some (small) enhancements

### PR DESCRIPTION
**Pull request by bremond**
mainly in those 2 functions:
     - set_dynamic_actors_visibility
     - set_position

     + split actors dictionay in dynamic_actors + static_actors

     Note: after looking again at some visualisations on chute con rocas
     there is no doubt that this is the multiple actors per object that
     does not scale (and finally this is obvious to me now).

### CODE REVIEW

**Comment by bremond** [comment]
Sorry, I made a new to branch to rebase everything and then I deleted vview_fixes branch in order to push the the rebased vview. Apparently, this made the previous PR disappear... So I understand that I should have created a new PR and close the previous one to keep the history.

File: io/swig/io/vview.py [hunk 1]
```diff
@@ -918,22 +890,36 @@ def init_cf_sources(mu):
         transforms[instance] = []
         offsets[instance] = []
         actors[instance] = []
```
Comment (radarsat1) [inline io/swig/io/vview.py]: Just need to remove this line and then it works for me. ("NameError: name 'actors' is not defined")

File: io/swig/io/vview.py [hunk 2]
```diff
@@ -918,22 +890,36 @@ def init_cf_sources(mu):
...
+        times_of_birth[instance] = io.instances()[instance_name].\
+                                   attrs['time_of_birth']
+
+        if instance > 0:
```
Comment (radarsat1) [inline io/swig/io/vview.py]: Is it "instance > 0" or "instance >= 0" for dynamic objects?

Comment (bremond) [inline io/swig/io/vview.py]: Argh.. actors[instance] should not exist as actors is splitted in dynamic_actors and static_actors, I miss this one when I rebased.
And yes, it is instance >= 0 for dynamic objects...

**Pull request merged**